arangod/Cluster/ClusterComm.cpp

```diff
@@ -1240,8 +1240,9 @@ std::pair<ClusterCommResult*, HttpRequest*> ClusterComm::prepareRequest(std::str
 }
 
 void ClusterComm::addAuthorization(std::unordered_map<std::string, std::string>* headers) {
-  if (_authenticationEnabled) {
-    headers->emplace("Authorization", _jwtAuthorization);
+  if (_authenticationEnabled &&
+      headers->find(StaticStrings::Authorization) == headers->end()) {
```

Comment:
the `find` here is probably superfluous. `emplace` will not overwrite an existing key.